src/main/java/core/basesyntax/supplier/FigureSupplier.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax.supplier;
+
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.Figure;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private final int radius = 10;
+    private final int number = 8;
+    private final int enumber = 4;
```

Comment:
what about constant names https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names And lets make more informative constant names